### PR TITLE
Add Cube Stack Umbrella Helm Chart

### PR DESCRIPTION
**Pull request by adnanrahic**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

https://github.com/cube-js/cube.js/issues/3337

### CODE REVIEW

File: examples/helm-charts/cubejs/README.md [hunk 1]
```diff
@@ -179,6 +179,7 @@ cubestore:
 | `global.devMode`                   | If true, enables development mode                                                                                               | `false` |
 | `global.debug`                     | If true, enables debug logging                                                                                                  | `false` |
 | `global.logLevel`                  | The logging level for Cube.js                                                                                                   | `warn`  |
+| `global.externalDefault`           | If true, uses Cube Store for storing Pre-aggregations                                                                           | `true`  |
```
Comment (lvauvillier) [inline examples/helm-charts/cubejs/README.md:182]: I'm not sure this is only for cubestore

File: examples/helm-charts/cubestack/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    apiPort: 4000
+
+    ## If true, enables development mode.
+    ##
+    devMode: false
+
+  redis:
+    ## The host URL for a Redis server
+    ## Naming this release "cubestack" will give you this default Redis URL 
+    url: redis://cubestack-redis-master:6379
```
Comment (lvauvillier) [inline examples/helm-charts/cubestack/README.md:55]: Maybe we can automatically configure this value if `redis.enabled` = true

File: examples/helm-charts/cubestack/values.yaml [hunk 3]
```diff
@@ -0,0 +1,810 @@
...
+commonLabels: {}
+
+## Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+
+#### Cubejs ####
+################
+cubejs:
+  enabled: true
```
Comment (lvauvillier) [inline examples/helm-charts/cubestack/values.yaml:20]: I think there is no need to copy all values here. You can just keep the `xxx.enabled` values

File: examples/helm-charts/cubejs/README.md (hunk 1 above)
Comment (adnanrahic) [reply examples/helm-charts/cubejs/README.md:182]: Nice catch! I'll edit this to mention other databases too.

File: examples/helm-charts/cubestack/README.md (hunk 2 above)
Comment (adnanrahic) [reply examples/helm-charts/cubestack/README.md:55]: Do you have an idea on how to do this?

**Comment by adnanrahic** [comment]
Closing in favor of: https://github.com/cube-js/cube.js/pull/3371

**Pull request closed without merging**